app/components/concept-scheme-uri-selector.js

```diff
@@ -11,14 +11,50 @@ export default class ConceptSchemeUriSelectorComponent extends Component {
 
   constructor() {
     super(...arguments);
+
     this.loadOptions();
   }
 
+  async getconceptSchemeByUri(uri) {
+    const conceptSchemes = await this.store.query('concept-scheme', {
+      filter: {
+        ':uri:': uri,
+      },
+    });
+    if (conceptSchemes && conceptSchemes.length >= 1) {
+      return conceptSchemes[0];
+    }
+
+    return null;
+  }
+
+  @action
+  async loadSelected() {
+    if (!this.args.forField.conceptSchemeUriOption) {
+      return;
+    }
+
+    const uri = this.args.forField.conceptSchemeUriOption;
+
+    if (uri) {
+      const conceptScheme = await this.getconceptSchemeByUri(uri);
+
+      this.setSelected({
+        uri: uri,
+        label: conceptScheme ? conceptScheme.label : uri,
+      });
+    }
+  }
+
```

Comment:
@ZijneDwaasheid  this resolves the issue that the already applied concept scheme is not showing in the dropdown when going to the configuration view

Reply:
this is now ok :)